jackson/src/test/java/com/baeldung/jackson/deserialization/nested/DeserializeWithNestedPropertiesTest.java

```diff
@@ -0,0 +1,63 @@
+package com.baeldung.jackson.deserialization.nested;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+public class DeserializeWithNestedPropertiesTest {
+
+	private String SOURCE_JSON = "{\"id\":\"957c43f2-fa2e-42f9-bf75-6e3d5bb6960a\",\"name\":\"The Best Product\",\"brand\":{\"id\":\"9bcd817d-0141-42e6-8f04-e5aaab0980b6\",\"name\":\"ACME Products\",\"owner\":{\"id\":\"b21a80b1-0c09-4be3-9ebd-ea3653511c13\",\"name\":\"Ultimate Corp, Inc.\"}}}";
+
+	@Test
+	public void testWhenUsingJacksonAnnotations_thenOk() throws IOException {

```

Comment:
Let's use the standard whenY_thenZ naming conventions for all the tests. For example, this one would become "whenUsingJacksonAnnotations_thenOk".